2022-05-17 08:40:18.967 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 08:40:24.000 | consumer_ts_server.py 69 | WARNING | 已连接， Waiting for queue: [ queue_new_scan_server ] routing_key: [ topic_new_scan_server ]
2022-05-17 08:44:05.606 | consumer_ts_server.py 46 | INFO | Received new message from - topic_new_scan_server
2022-05-17 08:44:05.607 | consumer_ts_server.py 48 | WARNING | Received 'hello world'
2022-05-17 08:44:05.607 | consumer_ts_server.py 53 | WARNING | Saving to DB
2022-05-17 08:44:05.607 | consumer_ts_server.py 59 | ERROR | 数据存储异常：Expecting value: line 1 column 1 (char 0)
2022-05-17 08:52:24.322 | consumer_ts_server.py 46 | INFO | Received new message from - topic_new_scan_server
2022-05-17 08:52:24.323 | consumer_ts_server.py 48 | WARNING | Received '{"ip": {"local_ip": "127.0.0.1", "public_ip": "127.0.0.1"}, "server_type": "\\u626b\\u63cf1", "status": "\\u6b63\\u5e38"}'
2022-05-17 08:52:24.323 | consumer_ts_server.py 53 | WARNING | Saving to DB
2022-05-17 08:52:24.323 | consumer_ts_server.py 59 | ERROR | 数据存储异常：'data'
2022-05-17 08:52:51.270 | consumer_ts_server.py 71 | WARNING | 已连接， Waiting for queue: [ queue_new_scan_server ] routing_key: [ topic_new_scan_server ]
2022-05-17 08:53:02.346 | consumer_ts_server.py 46 | INFO | Received new message from - topic_new_scan_server
2022-05-17 08:53:02.347 | consumer_ts_server.py 48 | WARNING | Received '{"ip": {"local_ip": "127.0.0.1", "public_ip": "127.0.0.1"}, "server_type": "\\u626b\\u63cf1", "status": "\\u6b63\\u5e38"}'
2022-05-17 08:53:02.347 | consumer_ts_server.py 53 | WARNING | Saving to DB
2022-05-17 08:53:02.429 | consumer_ts_server.py 59 | WARNING | Saved to DB
2022-05-17 08:53:39.127 | consumer_ts_server.py 46 | INFO | Received new message from - topic_new_scan_server
2022-05-17 08:53:39.127 | consumer_ts_server.py 48 | WARNING | Received '{"ip": {"local_ip": "127.0.0.1", "public_ip": "127.0.0.1"}, "server_type": "扫描1", "status": "正常"}'
2022-05-17 08:53:39.127 | consumer_ts_server.py 53 | WARNING | Saving to DB
2022-05-17 08:53:39.138 | consumer_ts_server.py 59 | WARNING | Saved to DB
2022-05-17 14:24:38.440 | consumer2.py 74 | WARNING | 已连接， Waiting for queue: [ queue_new_scan_server ] routing_key: [ topic_new_scan_server ]
2022-05-17 16:08:43.910 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：Transport indicated EOF
2022-05-17 16:08:51.053 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：StreamLostError: ("Stream connection lost: ConnectionResetError(54, 'Connection reset by peer')",)
2022-05-17 16:08:53.437 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:08:56.293 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:08:59.714 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:09:03.763 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:09:06.441 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:09:09.106 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:09:13.232 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:09:16.391 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:09:18.755 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:09:20.203 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:09:24.877 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:09:26.451 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:09:29.580 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:09:32.998 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:09:37.435 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:09:41.749 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:09:45.076 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:09:49.793 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:09:53.426 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:09:54.445 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:09:58.974 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:10:00.662 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:10:04.083 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:10:08.273 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:10:09.867 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:10:14.489 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:10:18.880 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:10:22.386 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:10:24.794 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:10:29.016 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:10:33.842 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:10:35.430 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:10:38.877 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:10:40.578 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:10:43.632 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:10:46.785 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:10:48.714 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:10:52.961 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:10:56.960 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:11:01.566 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:11:03.827 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:11:07.650 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:11:11.685 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:11:14.783 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:11:16.710 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:11:18.697 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:11:21.719 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:11:23.409 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:11:27.683 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:11:31.058 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:11:35.794 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:11:39.946 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:11:43.694 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:11:45.188 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:11:49.700 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:11:52.968 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:11:56.207 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:00.713 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:04.542 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:06.413 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:08.882 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:12.228 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:13.962 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:17.287 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:22.006 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:24.509 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:25.679 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:27.243 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:28.291 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:31.127 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:35.258 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:39.817 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:43.544 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:46.057 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:50.883 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:53.161 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:54.332 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:56.389 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:12:58.395 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:01.670 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:03.070 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:07.475 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:10.148 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:12.404 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:15.226 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:16.439 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:19.702 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:23.321 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:24.782 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:26.298 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:27.328 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:31.698 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:35.802 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:37.892 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:39.432 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:41.074 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:43.238 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:47.797 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:50.813 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:53.370 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:56.698 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:13:58.918 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:01.364 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:04.516 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:07.300 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:09.210 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:10.657 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:11.701 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:15.403 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:18.181 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:21.618 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:23.152 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:24.396 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:25.782 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:28.160 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:31.667 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:35.928 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:38.256 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:40.976 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:44.562 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:48.647 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:50.546 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:51.867 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:56.711 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:14:58.463 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:15:01.951 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:15:06.277 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:15:08.234 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:15:10.213 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:15:12.361 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:15:13.537 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:15:15.767 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:15:18.626 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:15:21.083 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:15:23.051 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:15:27.521 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:15:31.032 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:15:34.439 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:15:39.084 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:15:43.981 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:15:47.491 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:15:48.552 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:15:53.329 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:15:56.406 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:15:59.816 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:16:04.049 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:16:06.686 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:16:07.768 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:16:09.434 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:16:13.483 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:16:16.461 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:16:19.772 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:16:23.487 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:16:27.863 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:16:31.190 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:16:32.527 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:16:34.372 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:16:37.973 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:16:42.645 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:16:45.477 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:16:49.390 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:16:51.597 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:16:55.074 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:16:58.888 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:01.300 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:05.343 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:06.378 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:09.199 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:12.846 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:14.566 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:15.695 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:17.726 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:20.481 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:23.034 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:25.797 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:30.308 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:33.993 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:37.647 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:42.272 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:44.583 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:47.611 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:51.097 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:52.607 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:53.632 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:57.336 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:17:59.721 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:02.019 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:04.287 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:09.263 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:11.367 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:14.133 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:17.255 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:19.707 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:22.979 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:27.760 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:30.464 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:31.931 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:34.103 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:36.222 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:41.086 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:43.563 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:45.886 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:50.475 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:52.071 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:55.945 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:57.401 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:18:59.966 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:19:03.237 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:19:05.412 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:19:08.065 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:19:10.367 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:19:13.891 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:19:16.610 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:19:21.386 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:19:25.735 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:19:28.916 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:19:31.858 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:19:33.944 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:19:35.407 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:19:39.527 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:19:41.574 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:19:45.512 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:19:46.856 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:19:49.977 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:19:52.463 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:19:55.555 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:20:00.338 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:20:04.506 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:20:08.997 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:20:13.961 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:20:15.691 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:20:16.765 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:20:19.442 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:20:22.541 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:20:25.062 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:20:29.121 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:20:30.307 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:20:33.119 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:20:34.936 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:20:38.211 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:20:40.156 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:20:42.872 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:20:46.584 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:20:50.790 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:20:53.914 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:20:58.012 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:00.350 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:03.043 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:07.790 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:10.682 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:13.045 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:15.665 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:17.050 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:20.226 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:21.454 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:26.191 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:27.830 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:32.798 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:35.789 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:38.530 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:40.487 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:45.026 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:49.509 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:50.958 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:53.842 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:55.132 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:21:58.899 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:22:01.323 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:22:05.415 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:22:06.745 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:22:10.501 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:22:14.602 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:22:16.884 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:22:19.059 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:22:21.927 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:22:25.042 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:22:26.834 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:22:28.409 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:22:32.321 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:22:36.923 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:22:41.631 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:22:45.819 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:22:47.967 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:22:49.217 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:22:51.798 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:22:55.971 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:22:57.055 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:23:01.838 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:23:06.711 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:23:08.544 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:23:09.991 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:23:13.079 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:23:18.009 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:23:19.576 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:23:23.259 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:23:27.015 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:23:28.027 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:23:31.223 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:23:34.389 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:23:38.384 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:23:41.638 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:23:46.455 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:23:50.158 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:23:54.120 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:23:58.757 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:24:03.564 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:24:08.118 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:24:12.417 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:24:16.332 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:24:20.442 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:24:22.713 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:24:26.078 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:24:28.029 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:24:30.089 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:24:34.180 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:24:36.359 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:24:39.014 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:24:40.923 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:24:45.215 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:24:48.378 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:24:50.227 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:24:51.680 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:24:55.088 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:24:57.502 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:25:01.126 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:25:05.994 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:25:09.992 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:25:14.138 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:25:17.455 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:25:20.214 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:25:21.595 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:25:24.928 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:25:27.014 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:25:28.475 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:25:31.287 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:25:35.649 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:25:37.905 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:25:42.297 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:25:45.908 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:25:49.250 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:25:50.556 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:25:52.026 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:25:55.610 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:25:59.641 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:26:04.562 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:26:08.501 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:26:09.712 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:26:14.317 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:26:18.850 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:26:23.229 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:26:25.815 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:26:27.474 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:26:28.762 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:26:30.360 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:26:34.566 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:26:36.882 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:26:38.543 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:26:39.857 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:26:43.642 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:26:46.015 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:26:48.514 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:26:50.752 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:26:54.921 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:26:59.110 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:27:01.564 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:27:04.116 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:27:06.085 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:27:10.064 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:27:14.990 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:27:17.319 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:27:20.991 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:27:25.765 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:27:29.460 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:27:33.959 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:27:35.128 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:27:38.458 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:27:41.318 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:27:43.228 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:27:45.750 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:27:49.380 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:27:54.298 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:27:55.851 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:27:58.687 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:28:02.546 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:28:05.839 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:28:07.415 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:28:11.842 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:28:12.858 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:28:17.391 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:28:22.038 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:28:26.169 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:28:29.942 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:28:33.673 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:28:38.316 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:28:41.836 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:28:46.409 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:28:47.826 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:28:50.590 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:28:55.263 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:28:58.118 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:29:01.341 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:29:04.733 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:29:09.702 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:29:11.526 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:29:13.708 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:29:17.603 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:29:19.609 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:29:22.449 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:29:26.693 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:29:29.285 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:29:32.635 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:29:33.806 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:29:35.919 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:29:39.801 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:29:42.985 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:29:46.606 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:29:50.272 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:29:53.633 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:29:57.309 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:01.986 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:03.968 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:07.276 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:08.851 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:12.884 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:14.070 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:15.632 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:17.772 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:21.246 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:24.370 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:25.832 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:30.053 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:34.964 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:36.063 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:37.613 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:40.581 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:41.901 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:46.643 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:50.634 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:53.885 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:30:56.361 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:31:00.549 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:31:05.468 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:31:07.923 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:31:12.095 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:31:14.971 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:31:19.887 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:31:23.881 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:31:28.226 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:31:30.706 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:31:33.958 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:31:38.646 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:31:42.764 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:31:46.334 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:31:49.376 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:31:51.597 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:31:55.615 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:31:57.642 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:32:02.099 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:32:05.657 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:32:09.370 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:32:12.158 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:32:14.862 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:32:17.636 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:32:20.352 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:32:24.307 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:32:28.816 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:32:33.489 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:32:35.996 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:32:39.698 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:32:40.813 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:32:41.867 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:32:43.388 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:32:47.899 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:32:51.471 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:32:54.082 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:32:57.397 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:32:58.862 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:03.259 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:04.811 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:06.653 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:08.893 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:10.717 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:15.623 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:16.741 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:18.924 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:21.468 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:23.811 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:27.484 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:28.894 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:33.303 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:37.771 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:39.085 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:41.632 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:43.621 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:45.454 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:50.283 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:53.136 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:33:57.368 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:34:01.569 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:34:05.990 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:34:08.746 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:34:13.615 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:34:18.181 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:34:21.908 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:34:22.967 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:34:25.086 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:34:29.951 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:34:34.949 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:34:37.171 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:34:38.434 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:34:42.513 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:34:46.620 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:34:51.609 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:34:54.459 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:34:56.594 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:34:59.498 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:35:03.764 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:35:07.481 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:35:10.052 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:35:13.710 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:35:16.724 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:35:19.978 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:35:22.240 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:35:25.633 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:35:27.100 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:35:30.090 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:35:34.263 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:35:37.095 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:35:40.569 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:35:45.297 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:35:48.853 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:35:53.546 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:35:55.482 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:00.460 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:02.828 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:05.758 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:07.788 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:11.570 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:14.159 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:16.274 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:18.733 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:21.703 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:25.709 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:30.101 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:31.732 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:33.353 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:37.391 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:40.642 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:44.983 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:48.359 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:50.657 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:54.615 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:56.751 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:36:59.679 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:37:02.419 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:37:03.864 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:37:05.743 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:37:10.334 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:37:13.372 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:37:14.605 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:37:17.914 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:37:22.868 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:37:26.324 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:37:28.768 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:37:30.915 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:37:34.133 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:37:35.935 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:37:40.724 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:37:43.558 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:37:45.933 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:37:49.115 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:37:53.316 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:37:54.673 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:37:59.069 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:38:01.303 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:38:06.163 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:38:07.279 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:38:08.350 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:38:12.241 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:38:14.497 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:38:18.699 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:38:23.153 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:38:26.537 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:38:30.159 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:38:33.743 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:38:36.282 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:38:41.153 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:38:45.689 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:38:48.461 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:38:50.971 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:38:52.912 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:38:57.643 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:39:01.890 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:39:03.033 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:39:05.096 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:39:08.822 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:39:13.378 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:39:17.096 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:39:20.960 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:39:22.003 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:39:26.573 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:39:29.539 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:39:31.227 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:39:34.399 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:39:37.332 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:39:39.886 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:39:42.660 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:39:46.054 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:39:50.993 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:39:53.160 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:39:56.123 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:39:59.820 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:40:02.908 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:40:06.135 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:40:10.694 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:40:12.154 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:40:14.509 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:40:17.552 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:40:20.424 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:40:23.990 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:40:28.179 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:40:30.810 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:40:32.173 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:40:36.970 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:40:41.550 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:40:43.458 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:40:47.971 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:40:51.393 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:40:52.988 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:40:56.530 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:40:58.707 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:41:01.850 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:41:06.843 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:41:10.208 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:41:12.833 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:41:16.740 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:41:19.431 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:41:23.541 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:41:27.181 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:41:29.328 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:41:30.654 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:41:32.970 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:41:36.632 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:41:40.932 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:41:42.066 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:41:47.069 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:41:49.811 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:41:53.786 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:41:56.676 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:41:58.384 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:00.860 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:04.520 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:08.479 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:11.538 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:15.116 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:19.031 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:21.969 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:26.530 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:27.591 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:30.091 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:33.970 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:35.890 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:37.768 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:40.701 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:42.389 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:44.353 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:47.590 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:48.806 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:50.906 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:53.372 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:57.896 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:42:58.942 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:00.969 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:03.879 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:07.037 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:10.389 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:15.050 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:17.460 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:18.933 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:20.072 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:22.885 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:24.447 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:28.482 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:31.649 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:35.484 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:38.420 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:40.878 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:43.252 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:44.636 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:49.621 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:50.774 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:52.617 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:43:57.204 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:44:02.146 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:44:04.631 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:44:06.325 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:44:11.334 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:44:15.585 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:44:20.261 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:44:22.807 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:44:25.044 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:44:27.265 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:44:29.394 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:44:31.226 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:44:35.773 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:44:39.948 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:44:42.381 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:44:46.011 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:44:48.107 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:44:51.908 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:44:53.957 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:44:55.658 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:44:58.433 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:45:01.743 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:45:03.206 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:45:04.376 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:45:07.329 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:45:11.582 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:45:13.678 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:45:17.748 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:45:19.032 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:45:23.985 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:45:25.026 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:45:28.052 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:45:31.503 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:45:36.328 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:45:40.514 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:45:42.092 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:45:45.041 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:45:49.394 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:45:54.326 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:45:58.294 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:01.310 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:03.484 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:05.910 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:09.427 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:12.957 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:14.852 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:17.342 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:21.102 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:24.936 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:27.804 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:29.352 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:30.813 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:35.616 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:39.882 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:41.175 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:43.165 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:45.832 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:48.730 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:52.761 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:53.978 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:55.532 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:46:57.338 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:01.373 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:02.743 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:05.945 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:07.418 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:09.251 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:12.951 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:16.672 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:18.593 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:23.440 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:26.036 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:27.852 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:32.740 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:36.533 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:38.212 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:40.457 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:41.919 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:46.290 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:49.714 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:53.545 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:54.842 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:56.859 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:47:59.744 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:01.362 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:03.985 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:06.384 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:10.071 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:12.326 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:13.485 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:17.717 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:19.420 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:23.487 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:26.573 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:28.119 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:32.828 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:34.908 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:36.919 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:38.096 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:39.555 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:43.118 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:44.491 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:48.916 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:50.169 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:55.093 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:48:58.697 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:49:03.493 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:49:07.070 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:49:10.721 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:49:13.083 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:49:14.302 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:49:16.379 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:49:18.322 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:49:21.937 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:49:26.228 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:49:28.444 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:49:32.479 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:49:34.703 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:49:38.288 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:49:41.544 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:49:44.243 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:49:48.481 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:49:50.926 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:49:55.062 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:49:56.070 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:49:58.145 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:50:02.758 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:50:06.665 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:50:09.624 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:50:12.555 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:50:15.073 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:50:16.493 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:50:21.143 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:50:23.906 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:50:25.842 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:50:28.592 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:50:33.335 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:50:36.653 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:50:38.209 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:50:42.188 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:50:47.125 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:50:49.430 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:50:51.195 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:50:55.850 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:51:00.119 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:51:04.975 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:51:08.676 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:51:12.558 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:51:13.827 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:51:18.516 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:51:20.683 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:51:25.454 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:51:30.327 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:51:34.589 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:51:36.300 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:51:40.986 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:51:43.252 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:51:44.666 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:51:49.590 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:51:52.856 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:51:54.800 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:51:57.020 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:51:58.162 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:52:02.989 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:52:05.575 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:52:10.047 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:52:14.501 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:52:17.580 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:52:19.357 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:52:23.987 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:52:28.305 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:52:31.937 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:52:33.824 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:52:35.300 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:52:38.929 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:52:41.014 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:52:45.369 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:52:47.643 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:52:51.771 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:52:55.460 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:52:57.797 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:52:59.603 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:53:01.511 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:53:03.689 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:53:07.902 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:53:12.637 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:53:16.683 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:53:18.655 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:53:21.585 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:53:24.627 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:53:28.922 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:53:33.722 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:53:38.250 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:53:40.900 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:53:44.760 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:53:48.743 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:53:50.080 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:53:52.778 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:53:56.051 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:53:58.582 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:54:03.010 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:54:04.091 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:54:07.579 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:54:10.070 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:54:13.680 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:54:18.688 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:54:20.532 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:54:24.172 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:54:28.860 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:54:30.940 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:54:33.966 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:54:37.194 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:54:41.542 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:54:45.868 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:54:48.515 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:54:49.969 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:54:53.144 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:54:54.569 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:54:56.537 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:54:59.203 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:55:01.679 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:55:03.486 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:55:07.595 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:55:12.375 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:55:16.752 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:55:19.674 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:55:21.880 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:55:24.916 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:55:28.758 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:55:31.382 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:55:33.345 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:55:37.534 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:55:41.719 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:55:43.039 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:55:46.685 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:55:48.474 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:55:53.232 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:55:56.829 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:55:59.409 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:56:02.055 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:56:06.122 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:56:09.886 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:56:13.300 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:56:17.372 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:56:20.687 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:56:24.269 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:56:25.493 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:56:30.415 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:56:32.835 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:56:34.432 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:56:39.109 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:56:41.349 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:56:42.532 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:56:45.797 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:56:48.013 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:56:49.690 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:56:53.563 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:56:58.548 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:56:59.637 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:57:03.910 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:57:08.744 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:57:11.200 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:57:14.394 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:57:18.692 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:57:20.343 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:57:24.200 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:57:25.225 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:57:27.048 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:57:31.727 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:57:36.294 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:57:41.184 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:57:43.788 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:57:44.893 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:57:48.159 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:57:50.958 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:57:54.259 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:57:58.144 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:58:02.819 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:58:06.946 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:58:10.209 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:58:12.023 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:58:15.176 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:58:18.964 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:58:20.764 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:58:24.530 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:58:27.311 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:58:28.978 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:58:32.237 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:58:35.733 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:58:39.262 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:58:41.527 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:58:45.010 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:58:49.014 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:58:53.472 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:58:55.370 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:58:56.487 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:00.331 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:02.448 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:04.460 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:07.547 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:12.231 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:13.492 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:15.033 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:16.547 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:18.599 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:22.262 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:24.754 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:25.780 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:29.481 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:31.386 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:34.339 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:36.025 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:38.923 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:43.548 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:44.812 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:48.058 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:52.226 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:53.396 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 16:59:58.032 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:00.949 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:05.451 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:07.295 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:08.474 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:12.559 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:13.727 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:16.854 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:17.963 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:19.360 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:23.592 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:27.992 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:31.869 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:34.349 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:35.499 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:37.324 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:38.460 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:41.885 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:45.370 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:48.972 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:50.609 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:54.022 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:00:57.951 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:01:01.812 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:01:04.646 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:01:09.117 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:01:13.655 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:01:17.598 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:01:22.081 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:01:26.021 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:01:27.799 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:01:31.571 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:01:32.957 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:01:37.288 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:01:39.150 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:01:43.433 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:01:48.014 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:01:51.748 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:01:55.310 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:01:56.637 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:02:00.030 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:02:03.414 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:02:04.479 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:02:09.022 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:02:12.209 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:02:13.587 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:02:16.254 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:02:18.730 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:02:23.381 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:02:27.003 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:02:31.408 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:02:35.806 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:02:39.494 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:02:42.124 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:02:44.015 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:02:48.325 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:02:49.854 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:02:54.179 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:02:58.636 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:03:01.791 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:03:06.325 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:03:10.171 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:03:11.835 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:03:16.678 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:03:21.510 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:03:23.992 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:03:25.926 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:03:28.067 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:03:32.227 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:03:36.375 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:03:37.619 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:03:42.183 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:03:46.713 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:03:48.025 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:03:52.998 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:03:55.895 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:03:57.416 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:04:02.004 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:04:06.806 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:04:09.256 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:04:11.901 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:04:15.263 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:04:16.398 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:04:19.269 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:04:22.426 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:04:24.390 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:04:27.357 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:04:29.727 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:04:34.664 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:04:37.217 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:04:40.387 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:04:43.987 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:04:45.272 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:04:49.064 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:04:53.331 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:04:55.297 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:04:59.799 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:05:02.626 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:05:05.400 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:05:06.898 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:05:10.819 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:05:12.732 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:05:15.536 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:05:17.529 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:05:19.112 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:05:23.670 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:05:25.021 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:05:29.059 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:05:32.812 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:05:35.162 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:05:38.133 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:05:39.606 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:05:44.292 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:05:49.015 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:05:52.959 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:05:54.644 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:05:58.937 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:01.492 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:03.003 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:06.237 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:11.144 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:13.090 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:15.365 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:18.312 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:21.070 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:23.072 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:25.642 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:29.141 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:30.975 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:32.461 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:34.963 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:39.634 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:43.009 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:47.815 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:49.502 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:54.099 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:55.388 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:06:58.898 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:07:02.573 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:07:03.791 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:07:08.092 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:07:12.968 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:07:14.536 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:07:18.321 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:07:22.638 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:07:27.515 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:07:30.219 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:07:33.726 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:07:35.988 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:07:38.024 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:07:43.015 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:07:46.582 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:07:48.267 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:07:49.623 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:07:53.150 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:07:55.227 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:07:57.421 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:08:00.806 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:08:03.951 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:08:06.363 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:08:08.241 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:08:12.039 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:08:15.729 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:08:18.464 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:08:21.783 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:08:24.751 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:08:27.345 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:08:32.200 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:08:34.046 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:08:38.068 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:08:41.740 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:08:46.297 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:08:48.490 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:08:50.038 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:08:53.537 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:08:56.091 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:08:57.565 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:09:00.453 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:09:05.000 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:09:08.263 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:09:10.518 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:09:13.369 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:09:15.366 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:09:17.011 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:09:18.873 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:09:21.898 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:09:26.166 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:09:29.945 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:09:33.850 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:09:38.246 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:09:42.383 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:09:46.306 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:09:49.033 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:09:50.722 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:09:53.798 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:09:56.321 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:09:59.510 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:10:01.666 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:10:06.146 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:10:09.744 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:10:10.842 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:10:12.594 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:10:14.497 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:10:16.217 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:10:18.776 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:10:22.842 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:10:24.541 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:10:26.829 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:10:28.171 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:10:31.711 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:10:35.376 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:10:39.605 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:10:44.179 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:10:46.853 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:10:51.658 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:10:53.996 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:10:58.118 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:02.586 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:06.964 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:08.083 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:09.705 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:13.132 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:15.897 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:18.177 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:22.651 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:25.816 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:29.717 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:31.096 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:33.558 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:36.696 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:40.749 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:42.983 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:45.357 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:48.994 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:51.155 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:55.619 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:56.645 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:11:58.783 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:12:01.670 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:12:06.301 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:12:08.199 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:12:10.397 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:12:13.017 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:12:14.764 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:12:17.434 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:12:18.450 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:12:19.795 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:12:22.028 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:12:25.333 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:12:30.165 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:12:34.962 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:12:39.390 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:12:43.038 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:12:47.373 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:12:50.703 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:12:54.331 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:12:58.265 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:13:01.537 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:13:03.292 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:13:07.481 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:13:11.496 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:13:13.666 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:13:15.748 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:13:16.984 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:13:18.128 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:13:21.839 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:13:26.583 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:13:28.257 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:13:32.839 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:13:37.042 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:13:39.621 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:13:43.571 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:13:47.254 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:13:49.729 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:13:53.731 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:13:55.627 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:13:58.104 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:14:02.852 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:14:05.674 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:14:08.873 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:14:11.589 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:14:13.328 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:14:15.838 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:14:19.465 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:14:23.120 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:14:24.438 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:14:29.138 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:14:32.894 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:14:35.625 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:14:40.505 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:14:42.253 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:14:43.818 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:14:48.111 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:14:52.304 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:14:54.711 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:14:56.216 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:14:58.441 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:15:03.373 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:15:07.110 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:15:08.192 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:15:11.591 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:15:15.309 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:15:18.892 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:15:23.463 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:15:27.473 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:15:31.057 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:15:33.115 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:15:34.517 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:15:36.893 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:15:37.980 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:15:41.331 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:15:44.742 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:15:46.102 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:15:49.295 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:15:52.447 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:15:54.480 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:15:59.194 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:02.276 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:03.491 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:07.459 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:11.992 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:13.132 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:16.450 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:18.613 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:19.635 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:21.377 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:22.427 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:26.680 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:28.692 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:32.141 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:35.311 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:36.330 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:40.557 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:43.312 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:46.478 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:49.737 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:52.333 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:55.013 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:16:59.325 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:03.423 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:06.356 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:08.446 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:12.479 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:14.254 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:15.771 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:20.583 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:21.803 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:25.562 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:28.933 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:32.556 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:35.192 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:36.249 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:37.668 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:39.071 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:41.212 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:43.553 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:47.982 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:50.955 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:53.847 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:17:57.716 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:01.270 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:05.314 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:06.886 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:10.346 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:15.153 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:17.249 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:18.319 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:19.658 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:23.941 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:26.949 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:28.113 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:30.078 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:34.232 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:35.436 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:38.954 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:41.669 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:45.984 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:48.505 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:51.202 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:53.149 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:54.364 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:18:57.588 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:19:02.172 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:19:04.363 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:19:07.145 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:19:11.757 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:19:15.090 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:19:19.498 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:19:21.205 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:19:24.013 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:19:26.550 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:19:30.260 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:19:33.900 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:19:35.764 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:19:36.996 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:19:41.349 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:19:44.019 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:19:46.569 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:19:47.727 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:19:51.536 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:19:54.741 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:19:56.877 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:00.242 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:02.507 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:06.523 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:09.027 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:12.658 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:16.984 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:18.080 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:21.209 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:23.112 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:26.777 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:29.144 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:32.339 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:33.530 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:35.546 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:38.427 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:40.549 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:44.756 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:47.901 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:49.209 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:53.668 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:20:57.242 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:21:02.020 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:21:03.050 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:21:07.988 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:21:12.048 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:21:16.375 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:21:17.632 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:21:21.840 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:21:22.915 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:21:25.097 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:21:28.419 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:21:31.320 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:21:34.949 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:21:38.970 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:21:43.300 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:21:46.280 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:21:51.051 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:21:52.599 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:21:54.103 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:21:56.604 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:21:57.959 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:22:00.659 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:22:04.681 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:22:07.518 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:22:11.670 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:22:15.346 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:22:19.470 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:22:24.327 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:22:27.321 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:22:29.752 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:22:30.997 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:22:34.590 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:22:38.262 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:22:40.347 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:22:43.888 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:22:48.462 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:22:51.275 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:22:52.402 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:22:54.828 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:22:58.362 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:22:59.859 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:01.398 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:05.673 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:08.595 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:09.666 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:12.452 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:14.581 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:17.134 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:19.582 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:20.772 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:24.774 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:28.240 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:32.868 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:36.515 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:38.181 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:39.722 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:40.968 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:43.337 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:46.827 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:49.964 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:51.794 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:54.686 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:23:57.803 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:00.105 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:01.611 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:04.042 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:06.739 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:09.905 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:13.665 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:15.828 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:17.263 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:20.161 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:23.835 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:25.988 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:27.229 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:30.571 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:32.820 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:36.804 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:39.784 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:43.378 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:46.060 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:48.142 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:52.816 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:24:57.488 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:00.491 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:01.971 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:04.524 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:08.035 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:11.708 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:13.072 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:17.093 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:19.367 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:20.723 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:22.734 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:24.773 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:26.526 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:27.920 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:29.783 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:31.596 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:36.250 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:40.081 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:43.306 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:44.626 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:48.528 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:52.108 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:55.139 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:25:58.404 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:26:03.020 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:26:07.122 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:26:09.571 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:26:12.723 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:26:16.433 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:26:18.594 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:26:23.559 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:26:28.459 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:26:33.297 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:26:35.075 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:26:38.181 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:26:42.601 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:26:46.492 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:26:50.532 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:26:52.630 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:26:55.957 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:26:59.528 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:03.751 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:06.436 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:07.970 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:10.066 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:12.838 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:14.298 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:17.754 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:19.868 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:21.484 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:23.832 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:25.366 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:28.136 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:33.019 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:35.088 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:36.781 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:40.492 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:41.720 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:44.222 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:47.343 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:50.571 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:52.642 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:54.269 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:55.996 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:27:59.211 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:03.330 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:06.618 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:09.451 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:11.636 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:13.021 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:17.302 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:18.431 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:20.897 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:23.492 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:25.134 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:28.819 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:30.212 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:32.393 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:35.206 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:37.348 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:39.025 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:41.902 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:45.986 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:47.565 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:52.010 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:53.988 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:55.565 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:28:58.677 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:29:01.346 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:29:06.038 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:29:08.829 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:29:13.516 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:29:16.121 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:29:20.418 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:29:24.840 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:29:26.305 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:29:29.309 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:29:30.656 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:29:34.723 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:29:36.867 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:29:39.299 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:29:40.643 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:29:43.415 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:29:47.465 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:29:51.863 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:29:53.911 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:29:58.243 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:01.625 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:02.869 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:04.407 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:05.919 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:08.249 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:10.109 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:14.994 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:18.640 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:23.001 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:24.787 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:27.797 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:29.963 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:34.175 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:38.395 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:41.837 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:43.848 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:47.984 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:49.677 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:54.355 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:55.453 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:57.993 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:30:59.077 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:31:02.825 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:31:05.054 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:31:09.726 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:31:14.112 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:31:17.873 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:31:22.341 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:31:26.584 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:31:28.918 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:31:30.226 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:31:35.030 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:31:38.095 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:31:40.229 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:31:43.588 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:31:46.228 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:31:50.151 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:31:52.983 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:31:53.991 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:31:57.312 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:32:00.650 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:32:02.962 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:32:07.823 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:32:10.096 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:32:14.187 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:32:18.400 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:32:21.029 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:32:24.973 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:32:27.647 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:32:31.775 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:32:34.336 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:32:37.125 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:32:40.050 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:32:43.186 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:32:46.154 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:32:48.982 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:32:50.921 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:32:54.039 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:32:55.464 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:32:59.497 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:33:03.970 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:33:05.392 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:33:08.362 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:33:10.987 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:33:14.486 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:33:17.049 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:33:19.168 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:33:21.157 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:33:22.237 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:33:26.704 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:33:31.503 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:33:34.977 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:33:37.333 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:33:38.423 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:33:42.656 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:33:47.656 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:33:51.481 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:33:53.148 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:33:56.909 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:00.613 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:04.143 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:07.740 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:09.371 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:11.654 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:15.903 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:17.644 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:21.705 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:24.439 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:27.435 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:28.757 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:33.343 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:34.638 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:37.466 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:40.236 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:42.772 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:46.145 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:49.678 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:51.191 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:55.783 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:34:57.222 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:00.609 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:04.188 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:06.654 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:10.753 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:13.734 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:17.047 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:18.398 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:22.551 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:23.983 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:25.854 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:26.941 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:28.843 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:30.437 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:35.328 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:37.367 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:39.859 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:44.056 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:46.204 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:48.112 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:49.661 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:52.635 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:57.263 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:35:59.028 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:36:02.558 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:36:04.650 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:36:08.750 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:36:11.208 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:36:13.849 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:36:17.760 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:36:22.591 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:36:27.277 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:36:28.585 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:36:29.978 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:36:34.404 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:36:37.531 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:36:42.192 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:36:43.378 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:36:44.913 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:36:46.625 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:36:48.509 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:36:51.254 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:36:54.108 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:36:58.577 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:01.598 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:03.982 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:07.373 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:09.058 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:13.846 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:15.506 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:17.973 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:20.017 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:24.563 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:28.340 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:32.331 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:33.986 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:38.327 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:40.175 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:41.599 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:43.240 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:45.651 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:47.558 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:50.504 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:55.290 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:37:59.047 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:38:02.354 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:38:06.610 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:38:09.281 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:38:10.744 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:38:14.727 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:38:17.493 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:38:21.827 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:38:24.779 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:38:29.489 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:38:32.459 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:38:35.358 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:38:37.315 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:38:40.453 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:39:38.662 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:39:42.269 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:39:44.041 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:39:48.011 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:39:51.687 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:39:56.358 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:39:57.567 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:40:02.389 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:40:53.708 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:40:54.749 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:40:58.423 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:41:00.492 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:41:02.626 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:41:03.962 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:41:07.410 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 17:57:28.413 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 18:12:54.309 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 18:12:57.710 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 18:28:01.801 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 18:28:03.779 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 18:28:06.901 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 18:28:08.446 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 18:28:10.845 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 18:28:12.396 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 18:43:39.299 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 18:43:43.565 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 18:43:45.432 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 18:43:46.654 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 18:43:50.923 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 18:43:52.257 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 18:43:56.047 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 18:43:57.462 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 18:44:00.426 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 18:44:01.956 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 18:59:04.310 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:16:47.033 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:16:50.570 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:32:26.854 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:32:30.966 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:32:32.892 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:32:36.231 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:32:39.524 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:32:42.928 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:32:44.633 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:32:47.742 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:32:51.855 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:32:53.858 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:32:57.489 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:48:31.117 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:48:34.680 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:48:37.941 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:48:39.016 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:48:41.365 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:48:46.350 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:48:50.325 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:48:53.129 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 19:48:54.177 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 20:04:40.374 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 20:20:18.459 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 20:20:20.460 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 20:20:21.941 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 20:38:17.609 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 20:38:19.432 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 20:55:38.465 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 20:55:39.766 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 21:13:35.451 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 21:13:39.325 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 21:29:21.831 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 21:46:20.134 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 21:46:23.090 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 21:46:24.229 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 22:02:33.081 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 22:02:34.263 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 22:17:43.060 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 22:17:46.230 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 22:33:57.922 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 22:50:50.549 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:07:39.304 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:07:43.592 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:23:32.149 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:23:35.959 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:23:38.554 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:23:41.482 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:23:44.136 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:23:45.207 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:23:49.499 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:23:50.657 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:23:52.921 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:23:56.621 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:39:23.213 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:39:24.667 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:39:25.808 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:39:27.379 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:39:30.450 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:39:33.539 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:39:36.813 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:39:38.223 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:39:39.460 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:39:43.289 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:39:44.803 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:57:45.538 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
2022-05-17 23:57:49.161 | consumer_ts_server.py 93 | ERROR | 连接异常，重新连接：
